tests/safety/test_volkswagen_pq.py

```diff
@@ -60,15 +60,16 @@ def _set_prev_torque(self, t):
   def _speed_msg(self, speed):
```

Comment:
Not sure why PQ doesn't use the DBC to pack the values like other brands?

Reply:
PQ safety was upstreamed before the tests were converted to use the packers, and at the time they were converted, important parts of the matching DBC weren't yet upstream. It's been on my Round Tuit list to deal with, but fairly far down. If it's a blocker for your work, I can fix it in the next day or two.

Reply:
Agreed, will do.

Reply:
Opened an issue: #762